cli/src/main/resources/META-INF/native-image/com.devonfw.tools.IDEasy/ide-cli/reflect-config.json

```diff
@@ -16,5 +16,21 @@
         "parameterTypes": []
       }
     ]
+  },
+  {
+    "name": "com.devonfw.tools.ide.npm.NpmJsonObject",
+    "allDeclaredConstructors": true
```

Comment:
is this a new feature not supported by our outdated version of graalvm?
Do we need to update graalvm version?

Or should we manually configure the exact constructor?
I actually wanted to avoid all this. This is all very fragile stuff that is ugly on maintenance.
I would like graalvm to automatically generate all this from introspection of our JUnits.
AFAIK exactly that is possible and happening when the JUnits are executed together in the Graalvm build, but it is also not working for me...